README.md

```diff
@@ -304,7 +304,31 @@ try (DatabaseReader reader = new DatabaseReader.Builder(database).build()) {
     System.out.println(response.isResidentialProxy()); // false
     System.out.println(response.isTorExitNode()); //true
 }
+```
+
+### Anonymous Plus ###
+
+```java
+// A File object pointing to your GeoIP2 Anonymous IP database
+File database = new File("/path/to/GeoIP-Anonymous-Plus.mmdb");
 
+// This creates the DatabaseReader object. To improve performance, reuse
+// the object across lookups. The object is thread-safe.
+try (DatabaseReader reader = new DatabaseReader.Builder(database).build()) {
+    InetAddress ipAddress = InetAddress.getByName("85.25.43.84");
+
+    AnonymousIpResponse response = reader.anonymousPlus(ipAddress);
+
+    System.out.println(response.anonymizerConfidence()); // 30
```

Comment:
Should this (and `networkLastSeen()`, `providerName()`) use the public accessors?
```suggestion
    System.out.println(response.getAnonymizerConfidence()); // 30
```

Reply:
Yes, sorry. That is what I get for making the code changes yesterday and writing the docs today without testing them.